### ISSUE TITLE
Add note about review procedure (no pinging on opening PR!)

### ISSUE THREAD

**Issue by oscargus** [sympy/sympy#23179]
Right now, there is quite a lot of (seemingly arbitrary) pinging about reviews. I think that we should add some note about this in the PR template.

This is for example what matplotlib writes (as part of the additional text the first time a PR is opened).

```
            If you have not heard from us in a while, please feel free to ping
            `@matplotlib/developers` or anyone who has commented on the PR.
            Most of our reviewers are volunteers and sometimes things fall
            through the cracks.
```

To me it is a bit annoying to get pings on subjects that I have no idea about. I also think that it may be that other potential reviewers do not review if it was asked for specific names. Finally, while this may increase the number of notifications, it is still better that all developers do get it (and that there is not review pinging on opening the PR).

**Comment by oscarbenjamin** [comment]
I agree. Perhaps rather than pinging we could suggest sending an email to the mailing list if a PR has not had any reply for over a month. Any messaging about this should also indicate that a contributor should respond to all comments before pinging.

Also I think it's good to give an explicit time frame to manage expectations. Sometimes there shouldn't be any ping and the author should just wait for someone to get around to giving a review.

**Comment by moorepants** [comment]
I'd rather see something that sets realistic expectations: i.e. that we are volunteers and getting a PR in takes somewhat indefinite patience. Here's my take on it:

> Almost all of the developers with merge rights to SymPy are volunteers. Getting a pull request merged generally takes great patience. Careful understanding of the developer guide, sufficient unit & doc tests, following the style guides, and writing clear documentation make it much easier for any reviewers. If you are struggling to get the attention of a developer with merge rights to review you pull request, remember that anyone can review your work. Find someone else that needs a review and trade reviews with them. There is finite time available from the core developers and the easier you make it for them, the sooner they will press merge. You can always ping @sympy/developers if all the above fails you to grab their attention.
